microsoft-edge/devtools-guide-chromium/whats-new/2023/03/devtools-111.md

```diff
@@ -0,0 +1,161 @@
+
+In previous versions of Microsoft Edge, to emulate different themes or vision deficiencies, you would use the **Rendering** tool to, for example, emulate Achromatopsia, which is when the user is unable to perceive any color.
+
+In Microsoft Edge 111, you can now access these theme and vision deficiency emulation controls directly from the **Device Mode** toolbar, in addition to using the **Rendering** tool.
+
+For example, to emulate high contrast mode and test how your web content responds:
+
+1. In the **Device Mode** toolbar, to the right of the **Throttling** dropdown list, click **Emulate Media Features** (the Eyedropper icon, ![Eyedropper icon](./devtools-111-images/eyedropper-device-mode-icon.png)).
+1. Select **forced-colors: active**:
+
+![Emulating high contrast mode directly from the Device Mode toolbar](./devtools-111-images/new-emulation-options-hc-mode.png)
+
+You can also emulate vision deficiencies by clicking the Eye icon next to the Eyedropper icon.  For example, to emulate blurred vision:
+
+1. Click **Emulate Vision Deficiency** (the eye icon, ![Eye icon](./devtools-111-images/eye-device-mode-icon.png)).
+1. Select **Blurred vision**:
+
+![Emulating blurred vision directly from the Device Mode toolbar](./devtools-111-images/new-emulation-options-blurred-vision.png)
+
+See also:
+* [Emulate mobile devices (Device Emulation)](../../../device-mode/index.md)
+* [Check for contrast issues with dark theme and light theme](../../../accessibility/test-dark-mode.md)
+* [Emulate vision deficiencies](../../../accessibility/emulate-vision-deficiencies.md)
+
+
+<!-- ====================================================================== -->
+## Performance tool unminifies file and function names when profiling production code
+<!-- if change heading: update link in what's new item that links to here -->
+
+<!-- Subtitle: As long as the Performance tool can find sourcemaps from Symbol Server, the Performance tool automatically presents the original file and function names in the flame chart. -->
+
+Microsoft Edge 99 added a new feature to the **Performance** tool called **Unminify**. This **Unminify** feature applied sourcemaps to a profile that was recorded with the **Performance** tool, and then downloaded the unminified profile to your machine. The **Unminify** feature is described in [Use your source maps to display original function names in performance profiles](../../2022/03/devtools.md#use-your-source-maps-to-display-original-function-names-in-performance-profiles) in _What's new in DevTools (Microsoft Edge 99)_.
+
+In Microsoft Edge 111, this feature has been removed, because the Performance tool now automatically provides the original file and function names while profiling production code, as long as source maps are loaded, which can be done reliably and securely with Azure Artifacts Symbol Server.
```

Comment:
```suggestion
In Microsoft Edge 111, this feature has been removed, because the Performance tool now automatically provides the original file and function names while profiling production code, as long as source maps are loaded, which can be done reliably and securely by using Azure Artifacts Symbol Server.
```
not hi pri (b/c "with" isn't confusing here), but per corp style guide:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with - "Don't use _with_ as a synonym for _using_ or _by using_. "

@Reezaali @zoherghadyali - re: overuse of 'with', which is more vague, and sometimes ambiguous, than "by using", which is more explicit.

Reply:
also could break into 2 sentences

Reply:
In Microsoft Edge 111, this feature has been removed, because the Performance tool now automatically provides the original file and function names while profiling production code, as long as source maps are loaded.  Source maps can be loaded reliably and securely by hosting them on Azure Artifacts Symbol Server.